src/shop_header/shop_header.jsx

```diff
@@ -7,8 +7,10 @@ const ShopHeader = ({ logo, path }) => {
         <div>
             <img width='1440' height='330' className='shop_header_back' src='/photos/shop_header_background.png' alt='header back' />
             <div className='shop_header_text'>
-                {logo && <img width='77' height='77' className='shop_header_icon' src='/photos/header_back_phot_icon.png' alt='header icon' />}
-                <h2 className='shop_header_header'>{path}</h2>
+                {logo && <img width='77' height='77' className={path === 'Blog' ? 'shop_header_icon_blog' : 'shop_header_icon'} src='/photos/header_back_phot_icon.png' alt='header icon' />}
```

Comment:
again I don't like that element's className depends on hardcoded `path` check. figure out what's the difference between this edge case and standard behavior and think about a prop name that would reflect that difference